doc/Forums.md

```diff
@@ -4,49 +4,29 @@ Forums
 * [Home](help)
 
 
-Friendica also lets you create forums and/or celebrity accounts. 
+Friendica also lets you create community forums and other types of accounts that can function as discussion forums, celebrity accounts, announcement channels, news reflectors, or organization pages, depending on how you want to interact with others. Management of these pages can be delegated to other accounts, or a parent account can be designated to easily toggle multiple identities.
```

Comment:
Please add a single new line after periods. It won't change the display since Markdown requires two new lines to change paragraph, but it makes future changes easier by reducing the size of the diff to only the sentences that were changed, not the whole paragraph.

Please check the rest of the added text for this rule.